jmxtrans-output/jmxtrans-output-kafka/src/test/java/com/googlecode/jmxtrans/model/output/kafka/EmbeddedZookeeper.java

```diff
@@ -0,0 +1,120 @@
+package com.googlecode.jmxtrans.model.output.kafka;
+
+import static com.jayway.awaitility.Awaitility.await;
+import org.apache.zookeeper.server.ServerCnxnFactory;
+import org.apache.zookeeper.server.ServerConfig;
+import org.apache.zookeeper.server.ZooKeeperServerMain;
+import org.apache.zookeeper.server.quorum.QuorumPeerConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.reflect.Field;
+import java.util.Properties;
+import java.util.concurrent.Callable;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Embedded Zookeeper server for integration testing
+ */
+public class EmbeddedZookeeper {
```

Comment:
I thought about creating a rule but:
- As Kafka depends on Zookeeper, Zookeeper must be started before Kafka and stopped after Kafka, I thinks it's clearer to tell it.
- Both Kafka and Zookeeper depend on a data folder I create with the TemporaryFolder rule

I will try if a RuleChain may do the job